calculator/lexer.py

```diff
@@ -1,4 +1,65 @@
-def lex(s):
+ops = ["+", "-", "*", "/"]
+ops_without_subtraction = ["+", "*", "/"]
+
+
+def is_numeric_char(char):
+    """
+    is_numeric_char checks if a char is a number or a decimal point.
+    Input: char 
+    Output: True or False
+    """
+    if char.isdigit() or char == ".":
+        return True
+    else:
+        return False
+
+
+def convert_string(s):
+    """Convert_string function converts a string to a float or a int, depending on whether it has a decimal point or not
+       Input: s (Type: String)
+       Output: float or int conversion of  "s"
+"""
+    if "." in s:
```

Comment:
nit: You could do `float(s) if '.' in s else int(s)`, but this is fine.

Reply:
Thanks